lib/praxis/restful_doc_generator.rb

```diff
@@ -285,7 +290,8 @@ def write_index( versioned_types )
 
         reportable_types.each do |type|
           index[version] ||= Hash.new
-          display_name = type.name.split("::").last
+          next unless type.respond_to?(:display_name)
```

Comment:
we might, but only on the display side...you can even have conflicts if you use the same string in the display_name of two mediatypes.